src/commands/config/subcommands/random-voice-channel-names/set-random-voice-channel-names-frequency-config.ts

```diff
@@ -1,48 +0,0 @@
-import type { Guild, VoiceBasedChannel } from 'discord.js';
```

Comment:
This is actually a file rename from `set-random-voice-channel-names-frequency-config.ts` -> `set-random-voice-channel-names-cron-config.ts` with some changes in the new file